### PR TITLE
Edit improvement

### PR DESCRIPTION
**Pull request by iocanel**
Resolves: #2636 

And also makes edit more usable.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by manusa — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #### Improvements
 
 #### Dependency Upgrade
+* edit methods, now accept visitors as vararg or lambda.
```
Comment (rohanKanojia) [inline CHANGELOG.md:10]: Umm, shouldn't this go in `Improvements` section?

Comment (rohanKanojia) [reply CHANGELOG.md:10]: ohk, sorry. Just noticed we bumped Sundrio to `0.23.1`

**Review by rohanKanojia — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (metacosm) [reply CHANGELOG.md:10]: I think it'd be nice to add examples of each new modalities so that users can see the improvements directly.

**Comment by manusa** [comment]
[merge]

**Pull request merged**